src/backend/distributed/deparser/deparse_table_stmts.c

```diff
@@ -208,6 +214,8 @@ AppendAlterTableCmdConstraint(StringInfo buf, Constraint *constraint,
 	}
 	else if (constraint->contype == CONSTR_EXCLUSION)
 	{
+		/* TODO: check if possible with ADD COLUMN */
```

Comment:
Postgres docs don't claim support for adding exclusion constraints by using ADD COLUMN subcommands, make sure that this is the case.